Makefile

```
@@ -0,0 +1,57 @@
+# Tauri Development Makefile
+# Common development tasks for the project
+
+.PHONY: help dev build clean lint format check test tauri-dev tauri-build
+
+# Default target
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (11).

(maxbodylength)